tests/Avalonia.Headless.XUnit.PerTest.UnitTests/Avalonia.Headless.XUnit.PerTest.UnitTests.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Headless tests use Avalonia.Headless.XUnit extensions, that were written for the old xunit.
https://github.com/Youssef1313/Avalonia/tree/1eee91600a5e6998de27dae88c466ac9ca73368f/src/Headless/Avalonia.Headless.XUnit
 
I think it should be fine to update whole Avalonia.Headless.XUnit extensions set to v3 as part of ours 12.0 release.

Meanwhile, Avalonia.Headless.XUnit might need to stay of older version in this PR.

Reply:
I switched to xunit 2 for these.